fuse/host.go

```diff
@@ -716,12 +736,16 @@ func (host *FileSystemHost) Unmount() bool {
 		mntp = c_CString(host.mntp)
 		defer c_free(unsafe.Pointer(mntp))
 	}
+	host.RUnlock()
 	return 0 != c_hostUnmount(host.fuse, mntp)
 }
 
 // Notify notifies the operating system about a file change.
 // The action is a combination of the fuse.NOTIFY_* constants.
 func (host *FileSystemHost) Notify(path string, action uint32) bool {
+	host.RLock()
```

Comment:
My comments in `Unmount` apply here as well.

Reply:
In comparison to `func Unmount`, I don't run into a race condition when running `go test -race`.